guidedmodules/models.py

```diff
@@ -930,10 +930,10 @@ def get_access_level(self, user, allow_access_to_deleted=False, recursive=True):
         return None
 
     def has_read_priv(self, user, allow_access_to_deleted=False):
-        return self.get_access_level(user, allow_access_to_deleted=allow_access_to_deleted) in ("READ", "WRITE")
+        return self.get_access_level(user, allow_access_to_deleted=allow_access_to_deleted) in ("READ", "WRITE") or self.project.has_read_priv(user)
 
     def has_write_priv(self, user, allow_access_to_deleted=False):
-        return self.get_access_level(user, allow_access_to_deleted=allow_access_to_deleted) == "WRITE"
+        return self.get_access_level(user, allow_access_to_deleted=allow_access_to_deleted) == "WRITE" or self.project.has_read_priv(user)
```

Comment:
since this is `has_write_priv()`, did you want this to be `self.project.has_read_priv()` or did you want `self.project.has_write_priv()`?

Reply:
Fudge. There's currently no `project.has_write_priv(user)` method available.

I think we merge the request and then create the segregated `has_write_priv`. I'm not sure how much more work there is to create that segregated privilege. I think we are being a bit lenient during the transition from 0.8.6 to 0.9.0 to give anyone with view permissions also edit permissions. The reason is we also have to create UI in order to set these permissions.

Reply:
Sounds good, I'll merge.  I made a placeholder issue to remind us to create `has_write_priv` and UI, when it's a good time: #719.